os/arch/arm/src/common/up_initialize.c

```diff
@@ -208,7 +208,7 @@ void up_initialize(void)
 	devnull_register();			/* Standard /dev/null */
 #endif
 
-#ifdef CONFIG_DEV_URANDOM
+#if defined(CONFIG_DEV_URANDOM)
```

Comment:
I don't understand what you do.
`#ifdef` and `#if defined` are perfectly same.

Reply:
@sunghan-chang You're right. I'll check again.